src/lib/Liblog/pbs_messages.c

```diff
@@ -393,6 +393,14 @@ char *msg_corelimit = "invalid value for PBS_CORE_LIMIT in pbs.conf, continuing
 char *msg_resc_busy = "Resource busy";
 
 char *msg_job_moved = "Job moved to remote server";
+char *msg_init_recovsched = "Recovered scheduler %s";
+char *msg_sched_exist = "Scheduler already exists";
+char *msg_sched_name_big = "Scheduler name is too big";
+char *msg_unknown_sched = "Unknown Scheduler";
+char *msg_no_del_sched = "Can not delete Scheduler";
+char *msg_sched_priv_exists = "Another Sched object also has same value for";
+char *msg_sched_logs_exists = "Another Sched object also has same value for";
```

Comment:
reply_badattr() function when aux code is also passed appends the attribute name to the error message.
Changed the error messages as follows 
char *msg_sched_priv_exists = "Another Sched object also has same value for its sched_priv directory";
char *msg_sched_logs_exists = "Another Sched object also has same value for its sched_log directory";
And now passing zero for aux code solved the issue.
Qmgr: c sched sc1 sched_priv=/tmp
Qmgr: c sched sc2 sched_priv=/tmp
qmgr obj=sc2 svr=default: Another Sched object also has same value for its sched_priv directory
qmgr: Error (15215) returned from server